src/main/java/org/wildfly/common/archive/JDKSpecific.java

```diff
@@ -12,17 +12,17 @@
 final class JDKSpecific {
     private JDKSpecific() {}
 
-    static ByteBuffer inflate(final Inflater inflater, final ByteBuffer[] bufs, final long offset, final int compSize, final int uncompSize) throws DataFormatException, IOException {
+    static ByteBuffer inflate(final Inflater inflater, final ByteBuffer[] bufs, long offset, final int compSize, final int uncompSize) throws DataFormatException, IOException {
```

Comment:
Not critical at all, but was it intentional for the `offset` to remove the `final` qualifier?

Reply:
Yeah, I pasted in the Java 9/11 overlay(s) verbatim as much as possible.